app/views/common/500.html.erb

```diff
@@ -1,21 +1,18 @@
 <%= alert :header => _("Oops, we're sorry but something went wrong"), :class => 'alert-danger',
   :actions => link_to(_('Back'), main_app.root_path, :class => 'btn btn-default'),
-  :text => h(exception.message),
+  :text => h(exception_message),
   :close => false %>
 
 <div class="pull-right">
   <%= documentation_button("7.3GettingHelp")%>
 </div>
+
 <p id="message">
   <%= _("If you feel this is an error with Foreman itself, please open a new issue with") %>
-  <%= link_to _("Foreman ticketing system"), "http://theforeman.org/projects/foreman/issues", :rel => "external" %>,
-  <%= _("You would probably need to attach the") %>
-  <%= link_to_function(_("Full trace"), "$('#backtrace').toggle()")%>
-  <%= _(", relevant log entries, and it is highly recommended to also attach the foreman-debug output.") %>
+  <%= link_to _("Foreman ticketing system"), "https://projects.theforeman.org/", :rel => "external" %>,
```

Comment:
I realized this may be because I suggested it in https://github.com/theforeman/foreman/pull/7220#discussion_r351908681, but I meant to say that `http://theforeman.org/projects` was moved to `https://projects.theforeman.org/projects` (quite long ago). However, it's still good to link to directly to the issues.

```suggestion
  <%= link_to _("Foreman ticketing system"), "https://projects.theforeman.org/projects/foreman/issues", :rel => "external" %>,
```

Reply:
This still applies, please change.